internal/mode/static/state/resolver/resolver_test.go

```diff
@@ -227,9 +289,9 @@ func TestIgnoreEndpointSlice(t *testing.T) {
 			ignore: false,
 		},
 		{
-			msg: "normal",
+			msg: "normal IPV6 address type",
```

Comment:
Should we also be testing when the endpoint address type isn't contained in allowedAddressTypes?

Reply:
I don't think this was addressed